llvm/utils/lit/tests/Inputs/shtest-env-positive/env-args-none.txt

```diff
@@ -0,0 +1,35 @@
+# Check default environment.
```

Comment:
Nit: for newer tests, I tend to recommend using `##` for comments, to help them stand out from lit/FIleCheck lines. Since you're splitting the old test, this effectively becomes a new test, so it's a good chance to do that bit of tidying up.

Reply:
Fixed.